projects/npm-tools/packages/js-insights/package.json

```diff
@@ -3,11 +3,11 @@
 	"dependencies": {
 		"@babel/parser": "^7.6.0",
 		"@babel/traverse": "^7.6.2",
-		"airtable": "0.8.1",
+		"airtable": "0.10.0",
```

Comment:
Maybe just kill this project... just flat out delete `js-insights`... my poor baby 😂

Reply:
Never!

![over-my-dead-body](https://user-images.githubusercontent.com/7074/94908768-7d664700-04a2-11eb-906f-33b069cc600c.gif)

In all seriousness though, it's not hurting us. If it ends up causing problems in the dependency graph we can archive it — ie. move it outside of the yarn workspace:

```
mkdir archived
git mv projects/npm-tools/packages/js-insights archived/
```

and twiddle the links that point at the old location. We can also skip linting etc of `archived/`. One of the strengths of the monorepo is that we never have to worry "losing" stuff, because we can keep it all together.